test/extensions/filters/network/http_connection_manager/config_test.cc

```diff
@@ -56,6 +56,12 @@ class HttpConnectionManagerConfigTest : public testing::Test {
   NiceMock<Tracing::MockHttpTracerManager> http_tracer_manager_;
   std::shared_ptr<NiceMock<Tracing::MockHttpTracer>> http_tracer_{
       std::make_shared<NiceMock<Tracing::MockHttpTracer>>()};
+  void httpConnectionManagerConfigHelper(
```

Comment:
I think this should be something like `createHttpConnectionManagerConfig`. WDYT?

Reply:
@dio Thanks for pointing it. May I force push now or wait for the ci test?